### ISSUE TITLE
[Bug]: Videos no longer play.

### ISSUE THREAD

**Issue by MartianWarMachine** [FreeTubeApp/FreeTube#6705]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

When trying to play any video all I get are the following errors.

"TypeError: Cannot read properties of undefined (reading 'text')"
&
"SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON"

Sorry if I'm not being clear I don't understand any of the technical stuff.
Been Using FreeTube since Oct. 2023. Don't recall having major issues before.

### Expected Behavior

The video plays?

### Issue Labels

API issue, accessibility issue, feature stopped working, content not loading, usability issue

### FreeTube Version

0.23.0 Beta

### Operating System Version

Edition: Windows 10 Pro Version: 22H2 Installed: 08/11/2021 OS build: 19045.5371

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Ferlic-Azid** [comment]
I also have this issue. I took a photo of the pop-ups that show when the video does not play.

![Image](https://github.com/user-attachments/assets/f6b41af5-2230-4715-8617-dcbd4252d04f)

**Comment by KunaPrime** [comment]
can confim the same issue for both .rm and .deb packages

**Comment by MartianWarMachine** [comment]
I don't understand. How do you fix the issue?
